packages/amplify_flutter/lib/categories/amplify_auth_category.dart

```diff
@@ -166,4 +165,38 @@ class AuthCategory {
         ? plugins[0].signInWithWebUI(request: request)
         : throw _pluginNotAddedException("Auth");
   }
+
+  /// Updates a single user attribute and returns a [UpdateUserAttributeResult]
+  Future<UpdateUserAttributeResult> updateUserAttribute({
+    @required String userAttributeKey,
+    @required String value,
+  }) {
+    var request = UpdateUserAttributeRequest(
+        userAttributeKey: userAttributeKey, value: value);
+    return plugins.length == 1
+        ? plugins[0].updateUserAttribute(request: request)
+        : throw _pluginNotAddedException("Auth");
+  }
+
+  /// Confirms a user attribute update and returns a [ConfirmUserAttributeResult]
+  Future<ConfirmUserAttributeResult> confirmUserAttribute({
```

Comment:
thanks for doc comment. I think a potential project would be to go around library and add these.

Reply:
Agreed. @haverchuck has [an open PR](https://github.com/aws-amplify/amplify-flutter/pull/397/files) that adds them to Auth but I think there are some open comments/questions.

Reply:
cool